README.md

```diff
@@ -309,6 +286,34 @@ Kubeclient::Client.new(
 )
 ```
 
+
+#### Google's Application Default Credentials
+
+On Google Compute Engine, Google App Engine, or Google Cloud Functions, as well as `gcloud`-configured systems
+with [application default credentials](https://developers.google.com/identity/protocols/application-default-credentials),
```

Comment:
Can we claim all these 4 scenarios (GCE, AppEngine, Functions, gcloud) should work out of the box?
I wasn't sure so wrote "if ... kubeconfig file has `user: {auth-provider: {name: gcp}}`" to be safe...

Reply:
All those systems are documented by Google to include application default credentials. See https://cloud.google.com/docs/authentication/production#obtaining_credentials_on_compute_engine_kubernetes_engine_app_engine_flexible_environment_and_cloud_functions and https://cloud.google.com/docs/authentication/production#obtaining_credentials_on_app_engine_standard_environment

Reply:
FYI, I think the link above should be updated to this: https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application